Specs/DataSources/KmlDataSourceSpec.js

```diff
@@ -93,6 +95,24 @@ defineSuite([
     var uberLabelColor = Color.fromBytes(0xee, 0xee, 0xee, 0xee);
     var uberLabelScale = 4;
 
+    var oldConsoleLog;
+    var consoleMessages;
+
+    beforeAll(function() {
+        oldConsoleLog = console.log;
+        console.log = function(msg) {
```

Comment:
I'm not quite sure what you mean by this. I know I can put spies on callbacks or events but how do I set it up to work with console.log?